### ISSUE TITLE
Hoverfly drops Reason-Phrase from responses

### ISSUE THREAD

**Issue by WouldYouKindly** [SpectoLabs/hoverfly#868]
Hoverfly drops HTTP Reason-Phrase from recorded responses in simulate mode. E.g. response from Gitlab API while in `capture` mode:

```
HTTP/1.1 404 Not Found
Cache-Control: no-cache
Connection: keep-alive
Content-Type: application/json
Date: Mon, 15 Jul 2019 20:28:20 GMT
Hoverfly: Was-Here
Server: nginx
Transfer-Encoding: chunked
Vary: Origin
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Request-Id: RTdg3duujd7
X-Runtime: 0.013190

{
    "message": "404 Project Not Found"
}
```

(Reason-Phrase here is `Not Found`)

Response from Hoverfly in `simulate` mode:

```
HTTP/1.1 404
Cache-Control: no-cache
Connection: keep-alive
Content-Type: application/json
Date: Mon, 15 Jul 2019 20:28:20 GMT
Hoverfly: Was-Here
Server: nginx
Transfer-Encoding: chunked
Vary: Origin
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Request-Id: RTdg3duujd7
X-Runtime: 0.013190

{
    "message": "404 Project Not Found"
}
```

This causes `aiohttp` Python library to fail, because it expects Reason-Phrase to always be present.

## Additional relevant information
Hoverfly version: 1.1.0

**Comment by tommysitu** [comment]
Thank you for the bug report, I've added a PR to fix it.

**Comment by tommysitu** [comment]
Fixed in [v1.1.1](https://github.com/SpectoLabs/hoverfly/releases/tag/v1.1.1)
